### DISCUSSION TITLE
Please add Twilio's Verify API as a service option for verifying OTP.

### DISCUSSION THREAD

**Discussion by ioucyf** [supabase/supabase#12589] [Feature Requests]
Please add Twilio's Verify API as a service option for verifying OTP. 

It's cheaper, does not require a number, and can be customized via twilio's dashboard if needed.

If that's all I needed for sign-up/sign-in/verify, then purchasing a phone number + using the Message service for lots of requests such as the ones mentioned is going to cost significantly more comparing to using the Verify API.

Please make it happen! 🙏🙏

**Comment by orichh** [comment]
this 10000%. hitting the same issue. especially with the changes requiring developers to do a2p 10dlc registration
